docs/BreakingChanges/141 WPF TreeViewItem must be used within a TreeView.md

```diff
@@ -0,0 +1,27 @@
+## ID: WPF TreeViewItem must be used within a TreeView
+
+### Scope
+Major
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Before 4.5, a TreeViewItem could be used outside of a TreeView. A change in 4.5 added a check during scrolling that assumes TreeViewItems is only used in the context of 
+TreeView. An example of when this is hit is if a TreeViewItem is the root of a DataTemplate or is used without being wihtin a TreeView.
```

Comment:
And because customers probably aren't familiar with VirtualizingStackPanels, we will likely want to explain that those are any 'list'-type controls (ListBoxes, DataGrids, ListViews, etc.) which have virtualization enabled.